cstar_ocean/case.py

```diff
@@ -304,9 +304,13 @@ def from_blueprint(
 
             match base_model_info["name"].casefold():
                 case "roms":
+                    from cstar_ocean.roms import ROMSBaseModel, ROMSComponent
+
                     ThisComponent = ROMSComponent
                     ThisBaseModel = ROMSBaseModel
                 case "marbl":
+                    from cstar_ocean.marbl import MARBLBaseModel, MARBLComponent
+
                     ThisComponent = MARBLComponent
                     ThisBaseModel = MARBLBaseModel
```

Comment:
Notice the `Case` now only imports roms- and marbl-specific classes when it needs them.

Reply:
Also FYI it goes against python conventions to use `CamelCase` for variable names, even if those variables are pointing to types. So it should be `this_component = ROMSComponent`. I only didn't change it in this PR to make it less confusing because I'm already changing a lot of names.